runtime/near-vm-errors/src/lib.rs

```diff
@@ -85,6 +85,14 @@ pub enum HostError {
     InvalidMethodName,
     InvalidPublicKey,
     ProhibitedInView(String),
+    TooManyLogs,
+    KeyLengthExceeded,
+    ValueLengthExceeded,
+    LogLengthExceeded,
+    NumberPromisesExceeded,
+    NumberInputDataDependenciesExceeded,
+    ReturnedValueLengthExceeded,
+    ContractSizeExceeded,
```

Comment:
Would be nice to include the limit amount and the actual value info into the errors